### ISSUE TITLE
System frameworks seem to be mangled. Is this correct?

### ISSUE THREAD

**Issue by jtreanor** [CocoaPods/cocoapods-packager#58]
I am trying to build a package a library using packager but the project I add it to will not build due to errors that look like:
```
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_PodMyLibrary_CABasicAnimation", referenced from:
...
```
Correct me if I am wrong but does this mean that `CABasicAnimation` has been mangled? Why is this the case and is there something I am missing?

**Comment by jtreanor** [comment]
It may be worth adding that using `--no-mangle` causes these errors to go away, but I do want my dependencies to be mangled so this isn't a fix.

**Comment by neonichu** [comment]
Yes, the mangling implementation is very naive - see <https://github.com/CocoaPods/cocoapods-packager/blob/master/lib/cocoapods-packager/symbols.rb#L15>

The easy fix will be adding 'CA' there, but maybe it is time to come up with a smarter way of figuring out which classes should not be mangled.

**Comment by jtreanor** [comment]
Ah, that makes sense. I will have a look and see if I can come up with something.
I just discovered packager yesterday and I am impressed. Its a great tool :smile:

**Comment by neonichu** [comment]
Thanks, if you don't have a better way, a PR which simply adds that prefix would also be appreciated - that's at least better that the status quo :)

**Comment by jtreanor** [comment]
:+1:

**Comment by neonichu** [comment]
Fixed by #59
